aztfq/aztfq.go

```diff
@@ -8,17 +8,23 @@ import (
 	"strings"
 
 	"github.com/magodo/azure-rest-api-bridge/ctrl"
+	"github.com/magodo/azure-rest-api-bridge/mockserver/swagger"
 )
 
-func BuildLookupTable(input []byte) (LookupTable, error) {
+type Option struct {
+	// ImplicitArrayIndex makes the array index to be implicit, e.g. "foos.*.id" -> "foos.id".
+	ImplicitArrayIndex bool
+}
+
+func BuildLookupTable(input []byte, opt Option) (LookupTable, error) {
```

Comment:
Can this option be a pointer?
```suggestion
func BuildLookupTable(input []byte, opt *Option) (LookupTable, error) {
```

Reply:
why do we need a pointer here.. for `nil`?